manifests/broker.pp

```diff
@@ -15,7 +15,7 @@
     Service[$broker_service] -> Exec['migrate_pulp_db']
   } else {
     if $pulp::messaging_transport == 'qpid' {
-      package { 'qpid-tools': ensure => installed } -> Class['pulp::service']
+      Class['qpid::tools'] -> Class['pulp::service']
```

Comment:
I think this needs to be present even on a pulp server before migrations are run? I can do further testing to try to pinpoint this better.